altianalysis/run_difference_with_gpao.py

```diff
@@ -159,12 +180,20 @@ def parse_args():
     parser = argparse.ArgumentParser(description="Calcul de cartes de différences par rapport au RGE ALTI")
```

Comment:
détail : la description n'est plus à jour 
```suggestion
    parser = argparse.ArgumentParser(description="Calcul de cartes de différences entre 2 sources de modèles numériques, ou d'une source par rapport au RGE ALTI")
```